lib/core/test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -1791,8 +1799,37 @@ instance Arbitrary Word31 where
     arbitrary = arbitrarySizedBoundedIntegral
     shrink = shrinkIntegral
 
+instance Arbitrary AssetDecimals where
+    arbitrary = AssetDecimals <$> choose (1, 19)
+
+instance Arbitrary AssetMetadata where
+    -- TODO: We should add a proper arbitrary instance
+    arbitrary = AssetMetadata
+        <$> (pure "asset")
+        <*> (pure "An asset")
+        <*> (oneof [pure Nothing, pure $ Just "AST"])
+        <*> genMaybe (pure $ AssetURL $ fromJust $
+                parseURI "https://asset.url")
```

Comment:
FYI - I first tried using an existing `Aribitrary URI` instance, but it failed, because it generated `http` urls despite the swagger api promising it to be `https`. 

I'm not sure what the metadata server promises, but if it allows `http` urls, then our swagger spec promises too much...